### PR TITLE
[memgraph-2.5.1-release < add-memgraph-2.5.1-changelog] Add changelog

### PR DESCRIPTION
**Pull request by antoniofilipovic**
### Description

Changelog for Memgraph 2.5.1

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] Documentation improvements

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by vpavicic** [comment]
1.5.1? 2.5.1? 
Can you give me more info on what this is concerning about?

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+### Bug Fixes
+
+- Fixed bug on LOAD CSV where Memgraph would use much more RAM than needed to load whole CSV file.
```
Comment (vpavicic) [inline docs/changelog.md:9]:
```suggestion
- The LOAD CSV clause now uses less RAM to load a whole CSV file. 
```

**Comment by antoniofilipovic** [comment]
> 1.5.1? 2.5.1? Can you give me more info on what this is concerning about?

This is concerning the new Memgraph release, which will contain mostly bugfixes

**Comment by vpavicic** [comment]
>

**Comment by antoniofilipovic** [comment]
@vpavicic I updated the base branch to `memgraph-2.5.1-release`

**Pull request merged**